sim/README.md

```diff
@@ -122,7 +122,15 @@ An update which should be sent to all players and spectators.
 
   [sim-protocol]: https://github.com/Zarel/Pokemon-Showdown/blob/master/sim/SIM-PROTOCOL.md
 
-One message type that only appears here is `|split`. This splits the next six lines into `spectator`, `p1`, `p2`, `p3`, `p4`, and `omniscient` messages. The `p1`, `p2`, `p3`, and `p4` logs will have exact HP values only for the corresponding player, while the `spectator` log will not have exact HP values for either player, and the `omniscient` logs will have exact HP values for both.
```

Comment:
Incidentally, this information was incorrect, there are still only 4 streams on HEAD. ;)